code/modules/random_map/automata/diona.dm

```diff
@@ -1,6 +1,25 @@
 /turf/simulated/wall/diona/Initialize(mapload)
 	. = ..(mapload, "biomass")
 
+/turf/simulated/wall/diona/attack_hand(mob/user)
+	if(istype(user, /mob/living/carbon/alien/diona) || istype(user, /mob/living/carbon/human/diona))
```

Comment:
is_diona is fucking weird, so I didn't use it.

Reply:
You should use it anyway. From what I can see it works fine.